### PR TITLE
Grace

### PR DESCRIPTION
**Pull request by gracestapkowski**
Screenshot 2024-04-03 at 3 14 14 PM
Began profile page

### CODE REVIEW

**Comment by gracestapkowski** [comment]
Today I updated the sell form to conform to the new database schema and properly handle the optional fields

**Comment by gracestapkowski** [comment]
Most recent update: Added backend functionality that matches the new schema. Just need to add the subcollection

File: src/App.jsx
Comment (HongjieWang98) [inline src/App.jsx:1]: Please rebase the main branch and update App.jsx to account for changes from other PRs prior to your latest PR.

image

**Review by HongjieWang98 — COMMENTED** [review]
Nice work on the profile page! Please rebase the main branch and update App.jsx to account for changes from other PRs prior to your latest PR.

**Comment by HongjieWang98** [comment]
> Most recent update: Added backend functionality that matches the new schema. Just need to add the subcollection

Thanks for the update! Can you please include the update on the "Description" as well?

**Comment by HongjieWang98** [comment]
There is a minor error on the Profile page of double declaration, could you fix that?
image

**Review by HongjieWang98 — APPROVED** [review]
I imported/deleted some path to solve the errors.

**Comment by HongjieWang98** [comment]
> Today I updated the sell form to conform to the new database schema and properly handle the optional fields

Give a more detailed description by screenshots of Grace's work: 
Before:
image

After:
image

File: src/App.jsx [hunk 1]
```diff
@@ -30,6 +32,7 @@ function App() {
       <Providers>
         <PageWrapper>
           <Routes>
+            {/* <Route path="/applicationform" element={<ApplicationForm />} /> */}
```
Comment (kevinbae15) [inline src/App.jsx]: but can do this in another PR
```suggestion
```

File: src/pages/Sell/ListingPage.jsx [hunk 2]
```diff
@@ -20,16 +61,45 @@ export default function ListingPage() {
       });
 
       setListing(data);
-      // TODO upload to database
-      navigate('/sell/confirmation/');
-    }
+
+      try {
+        // const auth = getAuth();
+        addDoc(collection(db, 'items'), {
```
Comment (kevinbae15) [inline src/pages/Sell/ListingPage.jsx:67]: note the collection will be called `textbooks`

File: src/pages/Sell/ListingPage.jsx [hunk 3]
```diff
@@ -78,16 +153,24 @@ export default function ListingPage() {
         isLoading={isLoading}
         required
       />
+      <label>
+          Condition
+          <select value={answer} onChange={handleChange}>
+            <option value="poor">Poor</option>
+            <option value="fair">Fair</option>
+            <option value="good">Good</option>
+            <option value="very good">Very Good</option>
```
Comment (kevinbae15) [inline src/pages/Sell/ListingPage.jsx]:
```suggestion
            <option value="very_good">Very Good</option>
```

**Review by kevinbae15 — APPROVED** [review]

File: src/pages/Sell/ListingPage.jsx (hunk 2 above)
Comment (HongjieWang98) [reply src/pages/Sell/ListingPage.jsx:67]: Got it. I've noted this into the Progress Doc.

**Review by Emerson-Wenzel — APPROVED** [review]
